src/app/cli/src/init/client.ml

```diff
@@ -1789,90 +1789,82 @@ let add_peers_graphql =
                   } ) ) ) )
 
 let compile_time_constants =
-  let genesis_constants = Genesis_constants.Compiled.genesis_constants in
-  let constraint_constants = Genesis_constants.Compiled.constraint_constants in
-  let proof_level = Genesis_constants.Compiled.proof_level in
+  let open Command.Let_syntax in
   Command.async
     ~summary:"Print a JSON map of the compile-time consensus parameters"
-    (Command.Param.return (fun () ->
+    (let%map_open config_file = Cli_lib.Flag.config_files in
+     fun () ->
+       let home = Core.Sys.home_directory () in
+       let conf_dir = home ^/ Cli_lib.Default.conf_dir_name in
+       let genesis_dir =
          let home = Core.Sys.home_directory () in
-         let conf_dir = home ^/ Cli_lib.Default.conf_dir_name in
-         let genesis_dir =
-           let home = Core.Sys.home_directory () in
-           home ^/ Cli_lib.Default.conf_dir_name
-         in
-         let config_file =
-           match Sys.getenv "MINA_CONFIG_FILE" with
```

Comment:
Reading of the env variable is now part of the automated config file discovery within `Runtime_config.Json_loader.load_config_files`, i.e. taken into account